src/cmd/turnkey/pkg/wallets.go

```diff
@@ -202,6 +206,71 @@ var walletExportCmd = &cobra.Command{
 	},
 }
 
+var walletAccountExportCmd = &cobra.Command{
+	Use:   "export",
+	Short: "Export a wallet account",
+	PreRun: func(cmd *cobra.Command, args []string) {
+		if walletAccountAddress == "" {
+			OutputError(eris.New("--address must be specified"))
+		}
+
+		if EncryptionKeyName == "" {
+			OutputError(eris.New("--encryption-key-name must be specified"))
+		}
+
+		if exportBundlePath == "" {
+			OutputError(eris.New("--export-bundle-output must be specified"))
+		}
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		tkPublicKey := EncryptionKeypair.GetPublicKey()
+		kemPublicKey, err := encryptionkey.DecodeTurnkeyPublicKey(tkPublicKey)
+		if err != nil {
+			OutputError(eris.Wrap(err, "failed to decode encryption public key"))
+		}
+		kemPublicKeyBytes, err := (*kemPublicKey).MarshalBinary()
+		if err != nil {
+			OutputError(eris.Wrap(err, "failed to marshal encryption public key"))
+		}
+		targetPublicKey := hex.EncodeToString(kemPublicKeyBytes)
+
+		activity := string(models.ActivityTypeExportWalletAccount)
+
+		params := wallets.NewExportWalletAccountParams()
+		params.SetBody(&models.ExportWalletAccountRequest{
+			OrganizationID: &Organization,
+			Parameters: &models.ExportWalletAccountIntent{
+				Address:         &walletAccountAddress,
+				TargetPublicKey: &targetPublicKey,
+			},
+			TimestampMs: util.RequestTimestamp(),
+			Type:        &activity,
+		})
+
+		if err := params.Body.Validate(nil); err != nil {
+			OutputError(eris.Wrap(err, "request validation failed"))
+		}
+
+		resp, err := APIClient.V0().Wallets.ExportWalletAccount(params, APIClient.Authenticator)
+		if err != nil {
+			OutputError(eris.Wrap(err, "request failed"))
+		}
+
+		if !resp.IsSuccess() {
+			OutputError(eris.Errorf("failed to export wallet account: %s", resp.Error()))
+		}
+
+		exportBundle := resp.Payload.Activity.Result.ExportWalletAccountResult.ExportBundle
```

Comment:
non-blocking nit: do explicit type-assert on `Result` to avoid panic if the wrong kind of result is returned.

Reply:
not sure if this is currently possible...?

`invalid operation: resp.Payload.Activity.Result (variable of type *models.Result) is not an interface`